lib/utils/job-utils/command-parser.js

```diff
@@ -1023,6 +1024,58 @@ function commandParserFactory(Logger, Promise, _) {
         }
     };
 
+    CommandParser.prototype[ip] = function(data) {
```

Comment:
I recommend to add some sample data in each step during parsing, so that brings convenience for other user to read the code.